python/bugstr/__init__.py

```
@@ -315,71 +451,308 @@ def _maybe_send(payload: Payload) -> None:
+    threads = []
+    for relay_url in _config.relays:
+        def publish_to_relay(url):
             try:
-                client.add_relay(relay_url)
+                client = Client(Keys.generate())
+                client.add_relay(url)
+                client.connect()
+                client.send_event(event)
+                client.disconnect()
             except Exception:
                 pass
```

Ruff (0.14.13)

`try`-`except`-`pass` detected, consider logging the exception

(S110)